dynamic-config/config-cc/src/test/java/org/apache/servicecomb/config/client/TestParseConfigUtils.java

```diff
@@ -80,4 +81,26 @@ public void testNotifyItemsChangedNeedRefresh() {
     }
     Assert.assertTrue(status);
   }
+  
+  @Test
+  public void testMergeDimensionItems() {
+
+    boolean status = true;
+    Map<String, Object> application = new HashMap<>();
+	Map<String, Object> service = new HashMap<>();
+	Map<String, Object> version = new HashMap<>();
+	Map<String, Object> tag = new HashMap<>();
+
+	Map<String, Map<String, Object>> items = new LinkedHashMap<String, Map<String, Object>>();
+	items.put("application", application);
+	items.put("service@app", service);
+	items.put("service@app#version", version);
+	items.put("service@app#version!tag", tag);
+    try {
+      Deencapsulation.invoke(pc, "mergeDimensionItems", items);
+    } catch (Exception e) {
+      status = false;
+    }
+    Assert.assertTrue(status);
```

Comment:
the dimentions should be out of order and  you can add some key/values and , then check the merged result